src/services/wallets.service.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// TODO: Make this a map instead of a common object. It's safer.
```

Comment:
`TODO` on this PR or for later?

Reply:
To do later. This is a comment to reinforce this [`Use of externally-controlled format string` issue](https://lgtm.com/projects/g/HathorNetwork/hathor-wallet-headless/snapshot/ce8e08444593ddf2828ed6c4611edaeff4c3e528/files/src/controllers/index.controller.js?sort=name&dir=ASC&mode=heatmap#xb9f3e98ba428f368:1) raised by LGTM bot.